src/components/utils/rqb-inputs/value-editor.tsx

```diff
@@ -0,0 +1,45 @@
+/**
+ * Copyright (c) 2023, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+import { ValueEditorProps } from 'react-querybuilder';
+import { useParameterState } from '../../dialogs/parameters-dialog';
+import { PARAM_LANGUAGE } from '../../../utils/config-params';
+import React, { useCallback, useMemo } from 'react';
+import { getComputedLanguage } from '../../../utils/language';
+import { FieldType } from '../../dialogs/filter/expert/expert-filter-constants';
+import { MaterialValueEditor } from '@react-querybuilder/material';
+
+const ValueEditor = (props: ValueEditorProps) => {
+    const [languageLocal] = useParameterState(PARAM_LANGUAGE);
+    const countriesListCB = useCallback(() => {
+        try {
+            return require('localized-countries')(
+                require('localized-countries/data/' +
+                    getComputedLanguage(languageLocal).substr(0, 2))
+            );
+        } catch (error) {
+            // fallback to english if no localised list found
+            return require('localized-countries')(
+                require('localized-countries/data/en')
+            );
+        }
+    }, [languageLocal]);
+
+    const countriesList = useMemo(() => countriesListCB(), [countriesListCB]);
+
+    function getValues() {
+        return Object.keys(countriesList.object()).map((country) => {
+            return { name: country, label: countriesList.get(country) };
+        });
+    }
```

Comment:
I think it is better if using in combination with above useMemo to compute directly values instead of creating a function then calling every render